typescript/src/doc_functionality/search.ts

```diff
@@ -65,8 +65,18 @@ export function buildSearchIndexJSON(pages: Array<DocumentationPage>, groups: Ar
     for (let block of allBlocks) {
       if (block.hasOwnProperty("text")) {
         let textBlock = block as DocumentationPageBlockText
+        let text;
+
+        // encode html when block.type=Code
+        if (block.type === "Code") {
```

Comment:
Should it all be encoded? you can also write a>b and it will be dead :)

Reply:
Yeah, you're right. It will be safer. I changed it 👍 Thanks for approve!